webhook/__manifest__.py

```diff
@@ -1,12 +1,12 @@
-# -*- coding: utf-8 -*-
+# Copyright 2020 initOS GmbH.
 # Copyright 2016 Vauxoo - https://www.vauxoo.com/
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': 'Webhook',
-    'version': '10.0.1.0.0',
-    'author': 'Vauxoo, Odoo Community Association (OCA)',
+    'version': '11.0.1.0.0',
+    'author': 'initOS GmbH, Vauxoo, Odoo Community Association (OCA)',
```

Comment:
Mmh, I think this is more a contribution as changes are not subsequent. Add yourself only to contributors